components/bl-carousel-component/src/index.js

```diff
@@ -27,7 +28,10 @@ export default function Carousel({ component, eventHandlers }) {
   const [autoplay, setAutoplay] = useState(true);
   const autoplayRef = useRef();
 
-  const { nextImg, prevImg } = getAdjacentImages(currentImg, imagesList.length);
+  const { nextImg, prevImg } = useMemo(
+    () => getAdjacentImages(currentImg, imagesList.length),
+    [currentImg, imagesList]
+  );
```

Comment:
Not critical, but we use other codestyle everywhere

```suggestion
const { nextImg, prevImg } = useMemo(() => {
  return getAdjacentImages(currentImg, imagesList.length)
} ,[currentImg, imagesList]  );
```

So will be good to make it consistent.